pyannote/audio/pipelines/utils/diarization.py

```diff
@@ -74,8 +75,10 @@ def set_num_speakers(
 
     @staticmethod
     def optimal_mapping(
-        reference: Union[Mapping, Annotation], hypothesis: Annotation
-    ) -> Annotation:
+        reference: Union[Mapping, Annotation],
+        hypothesis: Annotation,
+        mapping_only: bool = False,
```

Comment:
I'd rather use `return_mapping` and later return a tuple `mapped_hypothesis, mapping` when `True`.